applications/cm-service/values.yaml

```diff
@@ -16,36 +36,112 @@ image:
   # @default -- The appVersion of the chart
   tag: null
 
-config:
-  # -- Logging level
-  logLevel: "INFO"
-
-  # -- Logging profile (`production` for JSON, `development` for
-  # human-friendly)
-  logProfile: "production"
-
-  # -- URL path prefix
-  pathPrefix: "/cm-service"
-
 ingress:
   # -- Additional annotations for the ingress rule
   annotations: {}
 
-# -- Affinity rules for the cm-service deployment pod
-affinity: {}
-
-# -- Node selection rules for the cm-service deployment pod
-nodeSelector: {}
-
-# -- Annotations for the cm-service deployment pod
-podAnnotations: {}
-
-# -- Resource limits and requests for the cm-service deployment pod
-# @default -- See `values.yaml`
-resources: {}
-
-# -- Tolerations for the cm-service deployment pod
-tolerations: []
+frontend:
+  # -- Number of web frontend pods to start
+  replicaCount: 1
+
+  # -- Resource limits and requests for the cm-service frontend pod
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "0.5"
+      memory: "500Mi"
+    requests:
+      cpu: "0.1"
+      memory: "230Mi"
+
+  # -- Affinity rules for the cm-service frontend pod
+  affinity: {}
+
+  # -- Node selector rules for the cm-service frontend pod
+  nodeSelector: {}
+
+  # -- Annotations for the cm-service frontend pod
+  podAnnotations: {}
+
+  # -- Tolerations for the cm-service frontend pod
+  tolerations: []
+
+worker:
+  # -- Number of worker pods to start
+  replicaCount: 1
+
+  image:
+    # -- Image to use for workers
+    repository: "ghcr.io/lsst-dm/cm-service-worker"
+
+    # -- Tag of worker image to use
+    # @default -- The appVersion of the chart
+    tag: null
+
+    # -- Pull policy for workers
+    pullPolicy: "IfNotPresent"
+
+  # -- Resource limits and requests for the cm-service worker pods
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "1"
+      memory: "1Gi"
+    requests:
+      cpu: "0.1"
+      memory: "550Mi"
+
+  # -- Affinity rules for the worker pod
+  affinity: {}
+
+  # -- Node selection rules for the worker pod
+  nodeSelector: {}
+
+  # -- Annotations for the worker pod
+  podAnnotations: {}
+
+  # -- Tolerations for the worker pod
+  tolerations: []
+
+  htcondor:
+    config:
+      # -- If specified, location for htcondor config file to be injected into
+      #    worker pod
```

Comment:
These hanging indents puts a weird whitespace in the auto-generated README.

Reply:
Hunh -- I copied that style from some other applications in phalanx.  I see what you are saying, though.  They don't show up when rendered in-browser at `lsst.io` because of the vagaries of HTML so I missed them!